.github/workflows/continuous-deployment.yaml

```diff
@@ -0,0 +1,43 @@
+name: Continuous Deployment
+# Based on https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-a-workflow-based-on-the-conclusion-of-another-workflow
+
+on:
+  # Trigger this workflow (continuous deployment) when the "Continous Integration" workflow
+  # had completed on the main branch.
+  workflow_run:
+    workflows: [Continuous Integration]
+    types: [completed]
+    branches: ["main"]
```

Comment:
Is there a reason we want this after contious intergration vs `on:[workflow_dispatch]` ?

Reply:
If we don't wait for CI to pass we might be deploying broken code